src/model/Series.ts

```diff
@@ -588,18 +604,33 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
             return;
         }
 
-        if (selectedMode === 'multiple') {
-            const selectedMap = this.option.selectedMap || (this.option.selectedMap = {});
+        if (selectedMode === 'series') {
+            this.option.selectedMap = 'all';
```

Comment:
Assign `this.option` to a temporal variable will have less code.

Reply:
Seems `getSelectedDataIndices` doesn't handle the case `selectedMap: 'all'` as I suggested.